### PR TITLE
Milestone 1 - Bootstrap

### PR DESCRIPTION
**Pull request by alexandermorales-dev**
In this milestone, we are implementing responsive design using Bootstrap predefined classes.

We have implemented few styles with CSS and applied basic structure with HTML

### CODE REVIEW

**Review by mricanho — CHANGES_REQUESTED** [review]
# 🔴  Changes Required 🔴  

Hello @AlexJustAlex2020 @DanSam5K, you did a great job in this milestone🔥, just a few changes are required:

- [x]  **In your README file add complete instructions about how to install the repository (git clone, cd, etc)** 
- [x] **In the header, the three sections(date, logo and subscribe) must be aligned**
- [x] **In the header the NW logo, must have a link**
- [x] **Right now the search field it's a span, it needs to be a form(input)**
- [x] **In mobile and tablet you need to show a hamburger button**
- [x] **In the footer the main navigation items must be links**

As described in the Code reviews limit [Policy ](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have 2 code reviews left for this project.

work hard

You can contact me on slack if you have any questions just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!👨🏽‍💻  🖥️

**Comment by DanSam5K** [comment]
Alright noted thanks for the feedback!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        </li>
+        <li class="flex-grow-1 nav-item">
+          <a class="nav-link" href="#">weather</a>
+        </li>
+        <li class="flex-grow-1 nav-item">
+          <input
+            type="text"
+            class="form-control form-control-sm border-0"
+            placeholder="search"
+          /><i class="fa fa-search top-fa" aria-hidden="true"></i>
```
Comment (descholar-ceo) [inline index.html]: As a previous reviewer pointed this out, you need to surround this input with a form tag, it should be a form

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @AlexJustAlex2020 and @DanSam5K  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone almost looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implement the requested changes from the previous review, keep it up, and your project is really almost done.
### The following is the list of what you have implemented so far (from the previous `review`)
:heavy_check_mark: In your README file add complete instructions about how to install the repository (git clone, cd, etc) :heavy_check_mark: 
:heavy_check_mark:   In the header, the three sections(date, logo and subscribe) must be aligned :heavy_check_mark: 
:heavy_check_mark: In the header the NW logo, must have a link :heavy_check_mark: 
:heavy_check_mark: :warning: Right now the search field it's a span, it needs to be a form(input) => But on this one you need to surround that input field with a form tag :warning: 
:heavy_check_mark: In the footer the main navigation items must be links :heavy_check_mark: 
#### :warning: However we need to address one remaining change to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's because this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] In mobile and tablet you need to show a hamburger button => This is implemented but not fully, while showing the humbarger bars, you need to make sure that the navbar looks like the original website's navbar, now the below screenshots I took them on the screens between 320px and 740px of width, kindly take your time and make sure that the hamburger and user icons are appearing well and they look like the original navbar  => that's what the previous reviewer meant you to do!

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-04-06 17-19-47 | Screenshot from 2021-04-06 17-18-20|

:point_right: Could you see the difference :question: :question: :question: 

:warning: Your commit messages should be in imperative mode or present tense, _i.e.:_ you should have written `Finish with first breakpoint(767px)` instead of writing `finishing with first breakpoint(767px)`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

___

_:warning: Reminder: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), when you reach to the limit of 3 code reviews and your project was not approved, you will need to repeat the project. => You are now left with only 1 chance to get a review on this project. So implement the requested changes very carefully. If you think that the code review was not fair, you can request a second-opinion using [this form](https://airtable.com/shrmBIv6cXYrq4Fgv)._
 
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by elisha2kyakpo1 — APPROVED** [review]
🎉💪✔Congratulations guys, You have made me proud by implementing everything the previous reviewers talked about. You can now sleep then come back and merge lol😂🤣.
In case you have a question, please reach me on slack, elisha kyakopo.
Cheer✔💪👈👍

**Review by DanSam5K — APPROVED** [review]

**Review by descholar-ceo — COMMENTED** [review]
# Status: Invalid Submission :x: :disappointed: :stop_sign: 
## Hello @AlexJustAlex2020 and @DanSam5K   :handshake: :handshake: :handshake: 

Thank you for submitting a code review request. And congratulations on that you have completed this milestone, however I am marking this code review request as invalid due to the following reasons:

- [ ] The submitted pull request is merged, you need to submit an open pull request, you will need to open another pull request which contain all of the changes you have done on this milestone

Happy coding!!!
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by elisha2kyakpo1 — COMMENTED** [review]
Hey guys, I can see you have worked hard so far, and your project is as good as the original💪
**There are few issues that have prevented you from moving forward, let's address them before we move**

1. make sure that all images have a link(clickable).
2. all titles have a heading and a link(clickable).
Please check all screenshots and make changes in all screen sizes.
clone-free signup 
original-free-signup

Please remove the margin on the desktop, check the original.
original-full

Fix the more stories part, the image and content are supposed to be side by side, check screenshot.
more stories
original-more stories

Fix my turn on a mobile screen, see screenshot
my-turn
original-my-turn
Note: free signup area should be worked on in all screen sizes, it does not look like the original.
Please include one missing part in column 2, (culture & travel).

When you make all the changes, submit another review.
**Remember** "As described in the Code reviews limit Policy you have 2 code reviews left for this project."
In case you have any questions concerning the review, please reach me on slack, elisha kyakopo
Happy coding🎉✔👈💪

**Pull request merged**